### ISSUE TITLE
Sesslon log save options naming

### ISSUE THREAD

**Issue by Nikolay-Po** [evgeniy1294/protodb#3]
- What is doing "Copy" option in right click menu of session log window? I think it worth to name it some more comprehensive.
- About "Copy as bytes", in my opinion it should be named "Copy as hex string" just because the result is not a "bytes" but they space-separated values.
- "Copy as string" is better to name "Copy as ASCII text" or what it is exactly do?

Thank you much for development!

P.S. I feel a lack of copy range selection. It will be convenient to me to make a selection of a part of log and to copy exact part of log into a clipboard.

P.P.S. "Copy" is actually "Copy text as displayed".

**Comment by evgeniy1294** [comment]
Yes, I change "Copy as bytes" to "Copy as HEX string" and "Copy as string" to "Copy as ASCII text".

**Comment by evgeniy1294** [comment]
The "Copy" action copies the message with the timestamp and channel. The proposed name may not correspond to the functionality. May be "Copy event text"?

**Comment by Nikolay-Po** [comment]
Hmm... "Copy event text" seems to be correct until you let the user select the range of characters to copy. May be: "Copy text as displayed"?
